CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to the Agoric SDK
 
-Thanks for getting involved!
+Thanks for getting involved! Code contributions are welcome, though if they affect the design please consult with the core team before investing much effort. The core team manages the project, so please do not make project management suggestions as they will distract from the work.
```

Comment:
I tried to understand 10 issues, and failed quite heavily. This is not because I'm dumb. This is because the project-management here has several deficits.

Blocking people from making suggestions is like saying "we manage things perfect".

Well, you are not.

Reply:
Hi @abebeos I'm curious... how did you find Agoric? What's your background? What area are you most interested to contribute to?

Reply:
(And I'm perfectly fine to back-off, but I really think you need folks of my kind.)

Reply:
@abebeos I do appreciate your enthusiasm and don't want to stifle it. We definitely don't manage things perfectly! That wasn't my claim. It's that Github comments about how to manage the project or setting priorities aren't help. We do a core team that has meetings and that talks through the day every day, so offering solutions through Github is going to lack a lock of necessary context.

There are trade-offs between project management and doing the engineering. We have struck a balance for our current priorities. If you have needs, let's hear them. Though I would request that you engage in a discussion forum (such as [Agoric Discord](https://discord.com/invite/qDW8DRes4s)) instead of the work tickets.

Reply:
@abebeos noted a strong aversion to discord earlier, so perhaps https://github.com/Agoric/agoric-sdk/discussions

Reply:
> We do a core team that has meetings and that talks through the day every day, so offering solutions through Github is going to lack a lock of necessary context.

This is part of the problem (public issue-tracker needs to reflect status/reality. Closed-door discussions need to feedback into public project resources)

There it is, can't help it, I suggest again...

> If you have needs, let's hear them. Though I would request that you engage in a discussion forum (such as [Agoric Discord](https://discord.com/invite/qDW8DRes4s)) instead of the work tickets.

I assess "Discord" as a toy (quite nice for gaming-related stuff), but not for professional work. How can one focus with these tons of emojs (not easy to block, btw.) etc.

> @abebeos https://github.com/abebeos/agoric-sdk/issues/1#issue-1410314715 a strong aversion to discord earlier, so perhaps https://github.com/Agoric/agoric-sdk/discussions

Yes, the "discussions" on github are a bit better and a good trade-off.

-

@turadg , you don't have to explain anything, in fact each sentence you write spawns new issues. But I will stay out of issues that are filed by you, and focus on one or two others, in order to finalize the 2nd part of my involvement here, which is to assess "Agoric" as a platform for use in personal/professional projects. (1st part was to find some bounties).

Just please do yourself (and Agoric) a favor, and do not merge this sentence:

> The core team manages the project, so please do not make project management suggestions as they will distract from the work.

Sounds quite alienating.

## Possible Solution

Something like this:

> The core team manages the project. Please avoid making project management suggestions on the issue-tracker, as this will distract the core-team from the work. You can discuss project related stuff (incl. your suggestions) on Agoric Discord, channel xyz, or the Agoric Discussions, category xyz"

## P.S.

having said all this: know that I have myself a **super-high sensitivity** when others "plumb around" in my Issues (especially those that I've not yet mastered, where the brain needs weeks or months for "ripening"). But then, I'm not **a public project, where security is a top-most priority (and in conclusion a verifiable, transparent development process)**.

Reply:
Sample: https://github.com/Agoric/agoric-sdk/discussions/6569

(this works actually very well)